test/test_preparation_and_conversion.py

```diff
@@ -293,12 +298,6 @@ def coefficient_of_performance_test(cutout):
 # %% Prepare cutouts to test
 
 
-TIME = "2013-01-01"
-BOUNDS = (-4, 56, 1.5, 62)
-SARAH_DIR = os.getenv("SARAH_DIR", "/home/vres/climate-data/sarah_v2")
-GEBCO_PATH = os.getenv("GEBCO_PATH", "/home/vres/climate-data/GEBCO_2014_2D.nc")
-
-
```

Comment:
This section was just moved to the top of the file, because I added a reference to `TIME` in one of the methods above.